hooli-data-eng/hooli_data_eng/defs/dbt/component.yaml

```diff
@@ -0,0 +1,34 @@
+type: hooli_data_eng.defs.dbt.component.HooliDbtComponent
+
+attributes:
+  groups:
+    - selection: "company_stats locations_cleaned sku_stats company_perf"
+      run_on_code_version_change: true
+
+    - selection: "weekly_order_summary order_stats"
+      partitioning: "weekly"
+      freshness_check_sensor: weekly_freshness_check_sensor
+
+    - selection: "orders_cleaned users_cleaned orders_augmented"
+      partitioning: "daily"
+
```

Comment:
wow I love this so much! I need to spend some time thinking about this a bit more like if this is how we'd want to expose things vs. not, but the idea of showing someone how they would add a new dbt selection via YAML (partitioned or no), along with adding a schedule, is really really compelling.